pom.xml

```diff
@@ -7,7 +7,7 @@
         <version>2.0.4</version>
     </parent>
     <artifactId>eiffel-remrem-semantics</artifactId>
-    <version>2.0.13</version>
+    <version>2.0.14</version>

```

Comment:
Since the commit adds a feature and new public methods the version should be bumped to 2.1.0.